examples/RM3/wecSimInputFile.m

```diff
@@ -9,6 +9,9 @@
 simu.solver = 'ode4';                   % simu.solver = 'ode4' for fixed step & simu.solver = 'ode45' for variable step 
 simu.dt = 0.1; 							% Simulation time-step [s]
 
+simu.nlHydro = 1;
+simu.pressureDis = 1;
+
```

Comment:
@akeeste Okay, I see what's going on now. Please do not modify the input files for the examples, that's what is causing the unexpected response for the RM3. You can change them locally to test your updates, but we don't want to push those changes to the repo. I should be able to resolve this issue by pushing to your branch.

Reply:
FYI, the tests are passing, so with this minor update to the input file we should be good to merge the PR. 

``` matlab
   25 Passed, 0 Failed, 0 Incomplete.
   373.901 seconds testing time.
```

Reply:
@akeeste can you let me push to your fork to make these changes?